### PR TITLE
Add patrent->parent (from LO)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8239,6 +8239,7 @@ pathes->paths
 pathnme->pathname
 paticular->particular
 paticularly->particularly
+patrent->parent
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `patron`

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8239,6 +8239,7 @@ pathes->paths
 pathnme->pathname
 paticular->particular
 paticularly->particularly
+patrent->parent, patron,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: and "patent"

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

**Pull request merged**